src/components/Property/RaiseWorkOrder/RaiseWorkOrderForm.js

```diff
@@ -13,6 +13,8 @@ import TradeContractorRateScheduleItemView from './TradeContractorRateScheduleIt
 import Contacts from '../Contacts/Contacts'
 import WarningText from '../../Template/WarningText'
 import { buildScheduleWorkOrderFormData } from '@/utils/hact/workOrderSchedule/raiseWorkOrderForm'
+import { IMMEDIATE_PRIORITY_CODE } from '../../../utils/helpers/priorities'
```

Comment:
just a nitpick , sorry for that 
we can use '@/utils....' like the way you do in `raiseWorkOrderForm.js` to stay consistent